### PR TITLE
Update ViewSelector.js to add functionality

### PR DESCRIPTION
**Pull request by ecomerford**
Hey @chutten, for this PR, I've:

- Added functionality to ViewSelector.js (it can now handle different selections).
- Updated the naming conventions throughout.
- Added a SummaryView.js component

Please review when you have a chance! Thanks!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -8,13 +8,27 @@ class App extends React.Component {
...
     this.state = {
-      metricQuery: "",
+      metric: "",
+      version: "",
+      channel: "",
     }
   }
 
-  handleInputChange = (event) => {
+  selectMetric = (event) => {
```
Comment (chutten) [inline src/App.js:17]: `selectMetric` and `selectVersion` are event handlers. Should they be prefixed with `on` or `handle` or something to make that easier to tell from their names? Or is it the usual React thing that things not on this.state are handlers?

File: src/Components/ViewSelector.js [hunk 2]
```diff
@@ -1,23 +1,36 @@
...
-          <NavItem eventKey="4" title="evolution">
-            Evolution
-          </NavItem>
-        </Nav>
+      <Tabs
+        defaultActiveKey={1}
+        id="view-tabs"
+      >
+        <Tab
+          eventKey={1}
```
Comment (chutten) [inline src/Components/ViewSelector.js:13]: Are we eventually going to want to assign these semantic values (like some enum or something descriptive?)

**Review by chutten — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (ecomerford) [reply src/App.js:17]: Yep, you're right! I'll fix the names of the event handlers today.

File: src/Components/ViewSelector.js (hunk 2 above)
Comment (ecomerford) [reply src/Components/ViewSelector.js:13]: They should have a more descriptive value. I'll work on that today, as well. Thanks @chutten!

**Pull request merged**